db/migrate/20150401061243_create_notifications.rb

```diff
@@ -0,0 +1,18 @@
+class CreateNotifications < ActiveRecord::Migration
+  def change
+    create_table :notifications do |t|
+      t.integer :course_id, null: false, foreign_key: { references: :courses }
+      t.integer :user_id, null: false, foreign_key: { references: :users }
+      t.string :action
+      t.string :activity_content
```

Comment:
I'm not sure if we should put activity and notification in one table.

@lowjoel Do you have any comments ?